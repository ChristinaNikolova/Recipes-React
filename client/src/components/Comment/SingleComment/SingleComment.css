.single-comment {
    background-color: #F7ECDE;
    margin: 20px;
    padding: 20px;
}

.single-comment-content{
    font-size: 15px;
    line-height: 1.5;
}

.single-comment-info {
    color: #54BAB9;
    margin-bottom: 15px;
    font-weight: bold;
}