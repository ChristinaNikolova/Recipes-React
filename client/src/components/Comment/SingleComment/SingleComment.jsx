import './SingleComment.css';

function SingleComment({ content, formattedCreatedOn, clientUserName }) {
    return (
        <article className="single-comment">
            <p className="single-comment-content">{content}</p>
            <hr />
            <div className="single-comment-info">
                <i className="fas fa-calendar-alt"></i> {formattedCreatedOn}
            </div>
            <p className="single-comment-info">
                <i className="fas fa-user"></i> {clientUserName}
            </p>
        </article>
    );
}

export default SingleComment;