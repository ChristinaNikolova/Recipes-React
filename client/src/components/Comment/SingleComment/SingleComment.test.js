import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SingleComment from './SingleComment';

it('should render SingleComment component', () => {
    const wrapper = shallow(<SingleComment />);
    expect(toJson(wrapper)).toMatchSnapshot();
});