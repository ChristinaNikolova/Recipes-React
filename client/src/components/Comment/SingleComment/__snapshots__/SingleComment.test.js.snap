// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SingleComment component 1`] = `
<article
  className="post post-content"
>
  <div
    className="single-comment-wripper"
  >
    <p
      className="comment-content"
    />
    <hr />
    <div
      className="info"
    >
      <i
        className="fas fa-calendar-alt"
      />
       
    </div>
    <div
      className="info"
    >
      <i
        className="fas fa-user"
      />
       
    </div>
  </div>
</article>
`;
